stock_inventory_lockdown/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================================
+Lock down locations during inventories
+======================================
+This module lets you lock down the locations during an inventory.
+
+Usage
+=====
+
+.. image:: stock_inventory_lockdown/static/images/move_error.png
+   :alt: Error message
```

Comment:
@mreficent This link does not work, Can you please take a look?

Reply:
There is no way to solve this. We can put it in a way that it works in github or in a way it works in Odoo, but not in both.